static/RELEASE_NOTES.md

```diff
@@ -2,7 +2,7 @@
 
 Here you can find description of changes we've built into each release. While we try our best to make each upgrade automatic and as smooth as possible, there may be some cases where you might need to do something to ensure the application works smoothly. So please read through the release highlights!
 
-## 4.0.0-beta.1 (current version)
+## 4.0.0-beta.2 (current version)
```

Comment:
Shouldn't the changes be add to this?

Reply:
There are no user visible (major) changes? Release page will have full list of changes.

Reply:
Ah okay, I guess we count "Fix Landing Page width (#1359)" as a bugfix then?